.github/workflows/build_linux_jax_wheels.yml

```diff
@@ -71,23 +77,35 @@ jobs:
         with:
           path: jax
           repository: rocm/rocm-jax
-          ref: ${{ inputs.jax_ref }}
+          ref: ${{ matrix.jax_ref }}
 
       - name: Configure Git Identity
         run: |
           git config --global user.name "therockbot"
           git config --global user.email "therockbot@amd.com"
 
+      - name: "Setting up Python"
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        with:
+          python-version: ${{ inputs.python_versions }}
+
       - name: Build JAX Wheels
+        env:
+          ROCM_VERSION: ${{ inputs.rocm_version }}
         run: |
-          pushd rocm-jax
+          ls -lah
+          pushd jax
           python3 build/ci_build \
             --compiler=clang \
             --python-versions="${{ inputs.python_versions }}" \
-            --rocm-version="${{ inputs.rocm_version }}" \
+            --rocm-version="${ROCM_VERSION:0:5}" \
```

Comment:
Why truncate here?

Reply:
In a lot of places, the build scripts and the ROCm setup assume that the build number is always going to be of the form `X.X.X`. I tried this with the `X.X.X+dev...` version and the build failed.